src/ORM/DataObject.php

```diff
@@ -1369,10 +1373,10 @@ protected function validateWrite()
      *
      * @throws ValidationException
      */
-    protected function preWrite()
+    protected function preWrite(bool $skipValidation = false)
     {
         // Validate this object
-        if ($writeException = $this->validateWrite()) {
+        if ($writeException = $this->validateWrite($skipValidation)) {
```

Comment:
Should be something more like

```php
$writeException = false;
if (!$skipValidation) {
    $writeException = $this->validateWrite();
}
```

Reply:
Any discussion about this can be in https://github.com/silverstripe/silverstripe-framework/pull/11202#discussion_r1571448957 since this only changes if that does.